### ISSUE TITLE
Auto function for enums raising an error, but it should not

### ISSUE THREAD

**Issue by sitnarf** [microsoft/pyright#145]
**Describe the bug**
When using the auto function for enum, it rises `Argument missing for parameter 'value'` error.

**To Reproduce**
```
from enum import Enum, auto
class CacheBehaviour(Enum):
    ALWAYS = auto()
    NEVER = auto()
    AUTO = auto()

```
**Expected behavior**
It should pass the test.

**VS Code extension or command-line**
pyright 1.0.37 CLI

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version.

**Comment by erictraut** [comment]
This is now implemented in version 1.0.38, which I just published.
